src/store.js

```diff
@@ -1,11 +1,13 @@
 import { configureStore } from "@reduxjs/toolkit";
 import createSagaMiddleware from "redux-saga";
 import rootSaga from "./rootSaga";
+import popularMoviesReducer from './features/popularMovies/popularMovieSlice';
 
 const sagaMiddleware= createSagaMiddleware();
 
 const store = configureStore({
     reducer: {
+        popularMovies: popularMoviesReducer,
     },
     middleware: [sagaMiddleware],
 });
```

Comment:
You need to call sagaMiddleware.run(rootSaga) in this case to start the execution of root saga at the end of the code as well as export default store.